.github/workflows/main.yml

```diff
@@ -15,33 +15,38 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - run: |
-        git fetch --prune --unshallow
-
-    - name: Checkout NuttX Documentation
+    - name: Checkout NuttX repo
       uses: actions/checkout@v2
       with:
         repository: apache/incubator-nuttx
-        fetch-depth: 1
         ref: master
-        path: nuttx/master
+        path: nuttx
     - uses: actions/setup-python@v2
       with:
         python-version: '3.8'
-    - name: Generate requirements.txt file
+    - name: Build docs
       run: |
-        cd nuttx/master/Documentation/
         pip3 install pipenv
-        pipenv lock -r > requirements.txt
-    - uses: ammaraskar/sphinx-action@master
-      with:
-        docs-folder: "nuttx/master/Documentation/"
+        mkdir -p target/docs
+        cd nuttx/Documentation
+
+        export NUTTX_VERSIONS=$(git tag -l nuttx-10*)
```

Comment:
GitHub does not fetch tags by default you have to add that

Reply:
Yep, just trying to fix that, thanks